packages/components/molecules/f-mega-modal/README.md

```diff
@@ -139,59 +92,20 @@ In addition to this, the modal exposes some classes which you can target in your
 | `open` | This event is emitted when the modal is opened. |
 | `close` | This event is emitted when the modal is closed. |
 
-You can add event listeners for these like so
-
-```vue
-<template>
-  <mega-modal
-    @open="onModalOpen"
-    @close="onModalClose">
-    <p>Modal content</p>
-  </mega-modal>
-</template>
-
-<script>
-export default {
-  methods: {
-    onModalOpen () {
-      // Do stuff here
-    },
-
-    onModalClose () {
-      // Do stuff here
-    }
-  }
-}
-</script>
-```
-
 ## Development
 
-Start by cloning the repository and installing the required dependencies:
-
 ```sh
 $ git clone git@github.com:justeat/fozzie-components.git
 $ cd fozzie-components
 $ yarn
 ```
 
-Change directory to the `f-mega-modal` package:
-
-```sh
-$ cd packages/components/molecules/f-mega-modal
-```
-
 ### Running storybook
 
-Storybook can be used to develop new and existing components.
-
-To start storybook:
-
 From the **root** directory run:
 
 ```sh
 $ yarn storybook:serve
 ```
 
-This will build and serve storybook at [http://localhost:6006](http://localhost:6006).
-
+Serve storybook at [http://localhost:6006](http://localhost:6006).
```

Comment:
I think this made more sense before potentially (as it now implies that you have to do something else to serve storybook at that URL, rather than the command above doing it).

Reply:
Maybe just changing this to `This serves storybook…`